docs/API.md

```diff
@@ -480,31 +480,42 @@ A single glob or array of globs that indicate which files to watch for changes.
 #### opts
 Type: `Object`
 
-Options, that are passed to [`gaze`][gaze].
+Options, that are passed to [`chokidar`][chokidar].
 
 #### fn
 Type: `Function`
 
 An [async](#async-support) function to run when a file changes.
 
-`gulp.watch` returns an `EventEmitter` object which emits `change` events with
-the [gaze] `event`:
+`gulp.watch` returns a wrapped [chokidar] FSWatcher object. If provided,
+the callback will be triggered upon any `add`, `change`, or `unlink` event.
+Listeners can also be set directly for any of [chokidar]'s events.
+
 ```js
 var watcher = gulp.watch('js/**/*.js', gulp.parallel('uglify', 'reload'));
-watcher.on('change', function(event) {
-  console.log('File ' + event.path + ' was ' + event.type);
+watcher.on('change', function(path, stats) {
+  console.log('File ' + path + ' was changed');
+  if (stats) {
```

Comment:
It would be a perf hit, and most users don't need/use it. Existing interface doesn't provide a `stat` object at all afaik, perhaps it just shouldn't be passed through here. 

Or perhaps *only* provide it if the user specified `alwaysStat`